vsintegration/src/FSharp.Editor/FSharp.Editor.fsproj

```diff
@@ -142,12 +153,11 @@
     <Reference Include="System.Xml" />
     <Reference Include="System.Xml.Linq" />
     <Reference Include="UIAutomationTypes" />
+    <Reference Include="VSLangProj, Version=7.0.3300.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a" />
```

Comment:
I have no clue why the full version and stuff was specified, but this was just through right-click > add reference.

Reply:
I think I figured out how to deal with this